syscount

```diff
@@ -151,8 +151,8 @@ if (( opt_count && opt_verbose )); then
 	fi
 
 	printf "%-6s %-16s %-17s %8s\n" "PID" "COMM" "SYSCALL" "COUNT"
-	perf script -f pid,comm,event | awk '$1 != "#" {
-		sub("sys_enter_", ""); sub(":", "")
+	perf script --fields pid,comm,event | awk '$1 != "#" {
+		sub("syscalls:sys_enter_", ""); sub(":", "")
```

Comment:
This is not mentioned in commit message but is required otherwise it prints like  syscallsmmap.